tdvt_odbc/tds/Staples.exasol_odbc.tds

```diff
@@ -5,7 +5,7 @@
   <connection class='federated'>
     <named-connections>
       <named-connection name='leaf'>
-        <connection tdvtconnection='exasol_odbctestname' class='exasol_odbc' odbc-connect-string-extras='' port='8563' schema='TESTV1' server='exasol.test.lan/15F9CA9BC95E14F1F913FC449A26723841C118CFB644957866ABB73C1399A7FF' tablename='Staples' username='TABLEAU_TEST_USER' />
+        <connection tdvtconnection='exasol_odbctestname' class='exasol_odbc' odbc-connect-string-extras='' port='8563' schema='TESTV1' server='exasol.test.lan' tablename='Staples' username='TABLEAU_TEST_USER' v-fingerprint='15F9CA9BC95E14F1F913FC449A26723841C118CFB644957866ABB73C1399A7FF' />
```

Comment:
Is it intended to commit the fingerprint here?

Reply:
Yes. The test framework requires configuring the DB connection here. As long as we can only run the tests manually it doesn't make sense to extract this.